R/dm.R

```diff
@@ -659,7 +673,44 @@ compute.zoomed_dm <- function(x, ...) {
 src_tbls.dm <- function(x) {
   # The x argument here is a dm object
   dm <- x
-  names(dm_get_tables(dm))
+  # FIXME: should this be forbidden for `zoomed_dm`? and should we not just take the names from `def`?
+  names(dm_get_tables_impl(dm))
+}
+
+#' @export
+copy_to.src_sql <- function(dest,
```

Comment:
This turns out to be a bad idea, because this method dispatches only on `dest` and currently doesn't use double dispatch. I'm hesitant to request this feature in dplyr, because it feels rather exotic.

Let's export `dm_copy_to()` again and not override `copy_to()`.